vmware/nsx/nsx_plugin.cat.rb

```diff
@@ -0,0 +1,813 @@
+name 'VMware NSX Plugin'
+short_description 'A RightScale Self-Service plugin for VMware NSX.'
+rs_ca_ver 20161221
```

Comment:
can you add a version in the Long Description

Reply:
Done